internal/stolonctl/pkg/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
+
+import (
+	"github.com/gravitational/trace"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// Client is the Kubernetes API client
+type Client struct {
+	*kubernetes.Clientset
+}
+
+// NewClient returns a new Kubernetes API client
+func NewClient(kubeConfig string) (client *Client, err error) {
+	config, err := getClientConfig(kubeConfig)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return &Client{
+		clientset,
+	}, nil
+}
+
+// Pods returns stolon pods matching the specified label
```

Comment:
`... matching specified filter`? I think `filter` is the wrong word (@knisbet correct me if I'm wrong) as it provides the opposite semantics - `label` or `selector` would be more apt.